### PR TITLE
Disabling HC and enabling DC for 101

### PR DESCRIPTION
**Pull request by thomasmaurel**

### CODE REVIEW

File: ensembl_prodinf/handover_tasks.py [hunk 1]
```diff
@@ -100,7 +100,7 @@ def handover_database(spec):
     spec['progress_complete']=0
     get_logger().info("Handling " + str(spec))
     submit_dc(spec, src_url, db_type, db_prefix, release, staging_uri, compara_uri)
-    submit_hc(spec, groups, compara_uri, staging_uri, live_uri)
+    #submit_hc(spec, groups, compara_uri, staging_uri, live_uri)
```
Comment (marcoooo) [inline ensembl_prodinf/handover_tasks.py]: I didn't tell you, but I would like to reverse things here: e.g. still submit HC in the background, but locking service on DC. Does that make sense ?

**Review by marcoooo — CHANGES_REQUESTED** [review]

File: ensembl_prodinf/handover_tasks.py (hunk 1 above)
Comment (thomasmaurel) [reply ensembl_prodinf/handover_tasks.py]: Sure, done in the new push.

File: ensembl_prodinf/handover_tasks.py [hunk 2]
```diff
@@ -198,12 +198,11 @@ def submit_hc(spec, groups, compara_uri, staging_uri, live_uri):
     try:
         hc_job_id = hc_client.submit_job(spec['src_uri'], cfg.production_uri, compara_uri, staging_uri, live_uri, None, groups, cfg.data_files_path, None, spec['handover_token'])
     except Exception as e:
-        get_logger().error("Handover failed, Cannot submit hc job")
-        raise ValueError("Handover failed, Cannot submit hc job {}".format(e))
+        get_logger().debug("Handover failed, Cannot submit hc job {}".format(e))
     spec['hc_job_id'] = hc_job_id
     task_id = process_checked_db.delay(hc_job_id, spec)
```
Comment (marcoooo) [inline ensembl_prodinf/handover_tasks.py]: If we were to "disable" HC locking, shouldn't we remove this statement from submit HC ?

**Review by marcoooo — CHANGES_REQUESTED** [review]

File: ensembl_prodinf/handover_tasks.py (hunk 2 above)
Comment (thomasmaurel) [reply ensembl_prodinf/handover_tasks.py]: This is just to have the error in the logs, it won't stop the handover

Comment (marcoooo) [reply ensembl_prodinf/handover_tasks.py]: Fine then :+1:

Comment (thomasmaurel) [reply ensembl_prodinf/handover_tasks.py]: Opps sorry, thanks for spotting this.

File: ensembl_prodinf/handover_tasks.py [hunk 3]
```diff
@@ -198,12 +198,11 @@ def submit_hc(spec, groups, compara_uri, staging_uri, live_uri):
...
     get_logger().debug("Submitted DB for checking as " + str(task_id))
-    return task_id
+    return
```
Comment (luca-drf) [inline ensembl_prodinf/handover_tasks.py]: Redundant return statement.
```suggestion
```

Comment (thomasmaurel) [reply ensembl_prodinf/handover_tasks.py]: I though a function need a return

Comment (luca-drf) [reply ensembl_prodinf/handover_tasks.py]: Python's functions implicitly return `None` if no `return` statement is encountered

**Review by marcoooo — APPROVED** [review]

**Review by luca-drf — APPROVED** [review]

**Pull request merged**